tests/duckdb/mod.rs

```diff
@@ -89,11 +87,11 @@ async fn arrow_duckdb_round_trip(
 #[case::timestamp(get_arrow_timestamp_record_batch(), "timestamp")]
 #[case::date(get_arrow_date_record_batch(), "date")]
 #[case::struct_type(get_arrow_struct_record_batch(), "struct")]
-#[ignore] // Decimal256(76,10) is not yet supported for ArrowVTab
+#[ignore] // External(UnableToCreateDuckDBTable { source: DuckDBFailure(Error { code: Unknown, extended_code: 1 }, Some("Invalid Input Error: Data type \"Decimal256(76, 10)\" not yet supported by ArrowVTab")) })
```

Comment:
This is still not working, TODO for @sgrebnov - update error messages based on new logic